src/Async.js

```diff
@@ -85,6 +86,9 @@ const Async = React.createClass({
 				cache: initCache(nextProps.cache),
 			});
 		}
+		if (nextProps.value !== this.props.value) {
+			this.loadOptions('');
```

Comment:
But if the input is `''`, then surely the async callback will just return the original limited result set, as expected? I.e. updating the query input. I don't see the problem since `this.loadOptions('')` is used elsewhere in the current prod code to do the same thing (e.g. [componentDidMount](https://github.com/alexjesp/react-select/blob/loadoptions-after-new-value/src/Async.js#L81)), maybe you could give an example?